### PR TITLE
Android/update versions gradle

### PR DESCRIPTION
**Pull request by shoatman**

### CODE REVIEW

**Comment by AdamBJohnsonx** [comment]
Do we want to upgrade the wrapper function via ./gradlew wrapper --gradle-version 6.6.1 ?  This will rewrite the batch files and some other generated code.

**Comment by shoatman** [comment]
@shahzaibj I'm waiting for the last travis build to complete (hasn't started yet)

File: .travis.yml [hunk 1]
```diff
@@ -1,52 +1,50 @@
...
-  - emulator -avd test -no-audio -no-window &
-  - android-wait-for-emulator
-  - adb shell input keyevent 82 &
+  - $ANDROID_HOME/platform-tools/adb wait-for-device
+  - $ANDROID_HOME/platform-tools/adb shell input keyevent 82 & 
 
 script:
-  - travis_wait 30 ./gradlew clean build connectedCheck
+  - cd $PWD
+  - travis_wait 30 ./gradlew clean build connectedCheck
```
Comment (shahzaibj) [inline .travis.yml:50]: can add new line at end of file

File: gradle.properties [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
+# This the TSL versionName...
+versionName=1.5.9
+
+# For OneAuth default abiSelection
+abiSelection=x86
```
Comment (shahzaibj) [inline gradle.properties:18]: new line ending

File: gradle/versions.gradle [hunk 3]
```diff
@@ -41,4 +42,24 @@ ext {
...
+    projectVersionName = "1.5.9"
+
+    junitLibraryVersion = "4.12"
+    androidSupportLibraryVersion = "28.0.0"
+    gsonLibraryVersion = "2.8.0"
+    supportTestLibraryVersion = "0.4.1"
+    mockitoLibraryVersion = "1.2"
+    dexMakerLibraryVersion = "1.2"
+    //#### The ABOVE are TSL version variables - END
 }
```
Comment (shahzaibj) [inline gradle/versions.gradle:65]: new line ending

**Review by shahzaibj — APPROVED** [review]

**Review by AdamBJohnsonx — APPROVED** [review]

**Pull request merged**